web/styles/subscriptions.css

```diff
@@ -378,6 +378,28 @@ h4.user_group_setting_subsection_title {
         }
     }
 
+    .left .no-streams-to-show,
+    .left .no-groups-to-show {
+        margin-top: calc(45vh - 75px);
+    }
+
+    .right .nothing-selected {
+        padding: 5px 5px 0;
+
+        .stream-info-banner a,
+        .group-info-banner a {
+            color: inherit;
+        }
+
+        .create-stream-button-container {
+            margin-top: calc(45vh - 128px);
```

Comment:
Needed these custom margins to align the text below button in right side to the text in left side. And the margin is different for streams and groups as height of banner is different due to different text length.

Ideally would be better to somehow avoid setting these margins.

Reply:
Messy, but it does seem non-broken. I bet it can be redone using `flex` without this, but it seems OK for now.